web/lib/handlers/event.js

```diff
@@ -0,0 +1,17 @@
+const mastermind = require('../mastermind');
+const Event = require('../models/event');
+
+const get = params => // eslint-disable-line no-unused-vars
+  mastermind([
+    // eslint-disable-next-line no-unused-vars
+    async (req, reply, cb) => {
```

Comment:
using async without a try catch is deprecated in node 

https://nodejs.org/dist/latest-v8.x/docs/api/deprecations.html#deprecations_dep0018_unhandled_promise_rejections

Reply:
it's caught by mastermind, which is an async series

Reply:
This https://github.com/CoderDojo/cp-zen-platform/blob/master/web/lib/mastermind.js?

I can't see how that would handle a rejected promise?

Reply:
See https://caolan.github.io/async/index.html "Using ES2017 async functions"

Reply:
thats was added in async 2.3 though this project still uses 0.9

Reply:
Good catch

Reply:
https://github.com/caolan/async/blob/v1.5.2/README.md#asyncify could work if you dont want to upgrade async  all the way to v2